ci-operator/step-registry/ovn/conf/ipsec-manifest/ovn-conf-ipsec-manifest-commands.sh

```diff
@@ -24,14 +30,13 @@ spec:
       ipsecConfig: {}
 EOF
 
-# additional os extension for 4.14+
-cp "${CLUSTER_PROFILE_DIR}/pull-secret" /tmp/pull-secret
-oc registry login --to /tmp/pull-secret
-ocp_version=$(oc adm release info --registry-config /tmp/pull-secret "${RELEASE_IMAGE_LATEST}" --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
-ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
-ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+# adapt to newer ipsec config for ocp versions >= 4.15
+if (( ocp_minor_version >= 15 && ocp_major_version == 4 )); then
+    /tmp/yq e '.spec.defaultNetwork.ovnKubernetesConfig.ipsecConfig.mode = "Full"' -i ${SHARED_DIR}/manifest_cluster-network-03-config.yml
+fi
 
-if (( ocp_minor_version >= 14 && ocp_major_version == 4 )); then
+# additional os extension for 4.14 only
+if (( ocp_minor_version == 14 && ocp_major_version == 4 )); then
     for role in master worker; do
```

Comment:
at the bottom of this loop can you add this line:

`cat ${SHARED_DIR}/manifest_${role}-ipsec-extension.yml`

Reply:
thank you